.circleci/config.yml

```diff
@@ -339,26 +388,44 @@ workflows:
   version: 2
   build-test-release:
     jobs:
+      - build-cache:
+          filters:
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
+      - style:
+          filters:
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
       - build-test-9.4:
           filters:
             tags:
               only: /v[0-9]+(\.[0-9]+)*/
+          requires:
+            - build-cache
```

Comment:
Do you think the pg jobs should also depended on the `style` job? Since they run in parallel maybe `style` could take a while. Maybe it'll be better for contributors to get a quick feedback if somehow they forgot to run the styler locally.

Reply:
We'll see about this later. PR it's looking really good as it is.

Reply:
Actually the styler runs quickly. So nevermind about this.